### PR TITLE
Testcase: Delete RBD PVC and provisioner while thick provisioning is progressing

### PR DESCRIPTION
**Pull request by jilju**
Test to delete RBD PVC and RBD provisioner leader pod while thick provisioning is progressing and verify that no stale image is present.
OCS-2533 - FT-Delete the pending RBD PVC while thick provisioning
OCS-2534 - FT-Delete RBD provisioner leader pod the pending RBD PVC while thick provisioning

### CODE REVIEW

File: ocs_ci/helpers/helpers.py [hunk 1]
```diff
@@ -475,6 +475,22 @@ def default_storage_class(
     return sc
 
 
+def default_thick_storage_class():
+    """
+    Return default RBD thick storage class
```
Comment (ebenahar) [inline ocs_ci/helpers/helpers.py:480]: add empty lines between docstring sections

File: tests/manage/pv_services/test_delete_pvc_while_provisioning.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+        pytest.param(
+            *[""],
+            marks=[polarion_id("OCS-2533"), tier2],
+        ),
+        pytest.param(
+            *["rbdplugin_provisioner"],
+            marks=[
+                polarion_id("OCS-2534"),
+                tier4,
+                pytest.mark.tier4a,
```
Comment (ebenahar) [inline tests/manage/pv_services/test_delete_pvc_while_provisioning.py:42]: you can use 'tier4a' from ocs_ci.framework.testlib

File: ocs_ci/helpers/helpers.py (hunk 1 above)
Comment (jilju) [reply ocs_ci/helpers/helpers.py:480]: Done

**Review by ebenahar — APPROVED** [review]

**Review by Avilir — APPROVED** [review]
LGTM

**Review by sidhant-agrawal — APPROVED** [review]

**Pull request merged**